frontend/src/pages/application/Settings.vue

```diff
@@ -71,26 +72,51 @@ export default {
     emits: ['application-delete', 'application-updated'],
     data () {
         return {
-            sideNavigation: [{ name: 'General', path: './settings' }],
+            sideNavigation: [{
+                name: 'General',
+                path: './settings'
+            }],
             input: {
                 projectName: this.application.name,
-                projectId: this.application.id
+                projectId: this.application.id,
+                application: this.application
+            },
+            editing: false,
+            options: {
+                instances: []
             },
-            editing: false
+            loading: {
+                instances: false
```

Comment:
you set values for this inside the `loadInstances` function, but I don't see where it's actually _used_ anywhere?